lib/Invoice.php

```diff
@@ -76,13 +76,10 @@ class Invoice extends ApiResource
     use ApiOperations\Retrieve;
     use ApiOperations\Update;
 
-    use ApiOperations\NestedResource;
+    const BILLING_CHARGE_AUTOMATICALLY = 'charge_automatically';
+    const BILLING_SEND_INVOICE = 'send_invoice';
 
-    /**
-     * Possible string representations of the billing reason.
-     *
-     * @see https://stripe.com/docs/api/invoices/object#invoice_object-billing_reason
-     */
+    const BILLING_REASON_AUTOMATIC_PENDING_INVOICE_ITEM_INVOICE = 'automatic_pending_invoice_item_invoice';
```

Comment:
I am not convinced I want that constant for now. Up to you, but I'd focus on just the core ones until we solve what to do for API versions in the future that remove values